src/mumble/NetworkConfig.cpp

```diff
@@ -191,13 +191,13 @@ void Network::prepareRequest(QNetworkRequest &req) {
 	if (g.s.bHideOS) {
 		req.setRawHeader(QString::fromLatin1("User-Agent").toUtf8(),
 						 QString::fromLatin1("Mozilla/5.0 Mumble/%1 %2")
-							 .arg(QLatin1String(MUMTEXT(MUMBLE_VERSION_STRING)), QLatin1String(MUMBLE_RELEASE))
+							 .arg(QLatin1String(MUMTEXT(MUMBLE_VERSION)), QLatin1String(MUMBLE_RELEASE))
```

Comment:
Shouldn't thid be
```suggestion
							 .arg(QLatin1String(MUMTEXT(MUMBLE_VERSION)), QLatin1String(MUMBLE_RELEASE_ID))
```
:thinking:

Reply:
I was confused as well:
https://github.com/mumble-voip/mumble/blob/ffb2b70b32558c4db0619ea22a7c6865fba58dc2/src/Version.h#L14-L18

Reply:
Hm. Maybe we should consider renaming `MUMBLE_RELEASE` to `MUMBLE_RELEASE_ID_STR`. Or if we're sure that we never need the non-stringified release ID, we could even make `MUMBLE_RELEASE_ID` a String directly and the change this code to
```
#ifndef MUMBLE_RELEASE_ID
#    define MUMBLE_RELEASE_ID "Compiled by User"
#endif

Reply:
I agree.